python/instrumentation/openinference-instrumentation-crewai/tests/openinference/instrumentation/crewai/test_instrumentor.py

```diff
@@ -60,38 +144,75 @@ def test_crewai_instrumentation(
     prompt_template_version: str,
     prompt_template_variables: Dict[str, Any],
 ) -> None:
-    if use_context_attributes:
-        with using_attributes(
-            session_id=session_id,
-            user_id=user_id,
-            metadata=metadata,
-            tags=tags,
-            prompt_template=prompt_template,
-            prompt_template_version=prompt_template_version,
-            prompt_template_variables=prompt_template_variables,
+    with using_attributes(
```

Comment:
Seems like these 2 tests could be combined into 1 no?

Reply:
i thought breaking it out into two is actually more readable  because of the nesting with the `using_attributes` with the `use_casette` context manager.

Reply:
Ah, I missed the `use_casette`, my bad